src/Homework4/task7.py

```diff
@@ -0,0 +1,16 @@
+'''
+Даны два натуральных числа. Вычислите их наибольший
+общий делитель при помощи алгоритма Евклида
+(мы не знаем функции и рекурсию).
+'''
+
+a = 14
+b = 170
+
+while a != 0 and b != 0:
+    if a > b:
+        a = a % b
+    else:
+        b = b % a
+
+print(a + b)
```

Comment:
Euclid algorithm doesn't expect to have result in sum. it's just a hack from your side not to figure out where 0 and where result.